tests/PHPUnit/PHPMatcherConstraintTest.php

```diff
@@ -50,6 +51,48 @@ public function test_it_sets_additional_failure_description() : void
         $this->assertFalse($constraint->evaluate(42));
     }
 
+    public function test_expected_as_string_is_sorted() : void
+    {
+        $constraint = new PHPMatcherConstraint('{"b": 2, "a": 1}');
+
+        try {
+            $constraint->evaluate('null');
+
+            $this->fail();
+        } catch (ExpectationFailedException $exception) {
+            $this->assertSame(
+                <<<'JSON'
```

Comment:
hmm I think this should be `<<<JSON`, it might be the reason why testsuite is failing?

Reply:
`<<<'JSON'` = `'$var will NOT be interpolated'` (nowdoc) and `<<<JSON` = `"$var will be interpolated"` (heredoc).
`php-cs-fixer` with current project's config automatically converts heredoc to nowdoc (`heredoc_to_nowdoc` rule) if there are no variables. In the same way it tries to convert `""` to `''`.

See https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.nowdoc for more info on Nowdoc. It's available since 5.3, so it should not fail the test...

Reply:
I found the problem: the comma after the closing now/heredoc tag should be on the next line before PHP 7.3. 
https://www.php.net/manual/en/migration73.new-features.php
Tests are green!